v5/core/client/network_configurations_options.go

```diff
@@ -0,0 +1,56 @@
+package client
+
+import "github.com/rollout/rox-go/v5/core/model"
+
+type NetworkConfigurationsBuilder struct {
+	GetConfigApiEndpoint string
+	GetConfigCloudEndpoint string
+	SendStateApiEndpoint string
+	SendStateCloudEndpoint string
+	AnalyticsEndpoint string
+	PushNotificationEndpoint string
+}
+
+type NetworkConfigurations struct {
+	getConfigApiEndpoint string
+	getConfigCloudEndpoint string
+	sendStateApiEndpoint string
+	sendStateCloudEndpoint string
+	analyticsEndpoint string
+	pushNotificationEndpoint string
+}
+
+func NewNetworkConfigurationsOptions(builder NetworkConfigurationsBuilder) model.NetworkConfigurationsOptions {
+	return NetworkConfigurations{
+		getConfigApiEndpoint: builder.GetConfigApiEndpoint,
+		getConfigCloudEndpoint: builder.GetConfigCloudEndpoint,
+		sendStateApiEndpoint: builder.SendStateApiEndpoint,
+		sendStateCloudEndpoint: builder.SendStateCloudEndpoint,
+		analyticsEndpoint: builder.AnalyticsEndpoint,
+		pushNotificationEndpoint: builder.PushNotificationEndpoint,
+	}
+}
+
+func (s NetworkConfigurations) GetConfigApiEndpoint() string {
```

Comment:
Should that be `GetGetConfigApiEndpoint`? 😛 😆

Reply:
well... maybe  🤷 😄  

the other endpoints doesn't have Get prefix to indicate we are getting the end point url..
do you think it will help future devs understand it better if it was `GetGetConfigCloudEndpoint`? then i'll change it..